_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+
+Since moving images say more than my words, please have a look at them:
+
+<video src="/img/folders.webm" class="no-flow img-responsive" width="220" height="140" autoplay loop></video>
+
+For those who prefer to read, this is for you: NewPipe now supports grouping subscriptions into folders. All the recent videos of those channels then appear in one re-loadable feed when you press on the folder. While [@mauriciocolli](https://github.com/mauriciocolli) implemented that, he also improved the `What's New` sidebar feed. With improved I mean finally fixing it: The videos are showing in the correct order of the upload, not the first video each channel in a random order. And on top of that, live streams have now a viewer count in those feeds!
+
+An additional feature squashed into this change is the implementation of a so called _fast-mode_. If you enable it in the `Content` menu, those feeds will load much faster from the services which provide a so called _dedicated feed_. The downside of that is that those feeds contain less information, which might result in duration or the live status of videos not shown in the preview. The decision is up to you: Living on the speeding edge and missing information or taking it nice and slow, but knowing **everything**.
+
+
+## Mute button
+
+This release doesn't stop at folders though. Thanks to [@karkaminski](https://github.com/karkaminski), NewPipe now has mute button. [@B0pol](https://github.com/B0pol) formed it further into the shape you can experience it in now. _"But where can I find it"_ you ask. Fear not, for I have prepared a gallery of screenshots. To access the pop-up and background player menus shown, you need to press on the notification from NewPipe.
+
+<img class="no-flow vertical" src="/img/screenshots/mute_main.jpg"/>
+<img class="no-flow img-responsive" src="/img/screenshots/mute_back.jpg"/>
+
+
+## Smaller Improvements
+
+[@comradekingu](https://github.com/comradekingu) took care of the English base strings in this release and made sure they are consistent. Further improving the user experience was also [@XiangRongLins](https://github.com/XiangRongLin) goal, he made sure that the seek duration jumped visibly to an even one when the fast inexact search option gets activated. Previously the user wouldn't see this directly, but experience when seeking for- or backwards. Thanks to [@opusforlife2](https://github.com/opusforlife2), two settings moved from `Appearance` to `Content`, where they belonged all along: `Show 'Next'` and `Content of main page`.
+
+[@wb9688](https://github.com/wb9688) and [@B0pol](https://github.com/B0pol) (in his second contribution) added support for more subdomains. Now `music.youtube.com` and `media.ccc.de` are natively supported by NewPipe, thanks you two. In the last point to this section, [@jonas-skywalker](https://github.com/jonas-skywalker) (may the force be with you) improved the search section. Now the magnifying glass vanishes once you start typing.
```

Comment:
```suggestion
[@wb9688](https://github.com/wb9688) and [@B0pol](https://github.com/B0pol) (in his second contribution) added support for more subdomains. Now `music.youtube.com` and `media.ccc.de` are natively supported by NewPipe. Thanks, you two. In the last point in this section, [@jonas-skywalker](https://github.com/jonas-skywalker) (may the force be with you) improved the search section. Now the magnifying glass vanishes once you start typing.
```

Reply:
media.ccc.de was supported before (and it is no subdomain, it's a whole service). New in this version is that one can share their URLs to NewPipe.

Reply:
oh okay.

Reply:
technically yes, but ccc.de is not a streaming / media service :D

Reply:
But MediaCCC is a distinct Service Newpipe supports. So whether or not the URL is a subdomain is not only irrelevant here, but also a bit misleading. I think there is not need to use the term at all. It's enough to say that MediaCCC and Youtube Music URLs can now be shared to Newpipe without error.

Reply:
I dropped subdomain now, is that alright?

Reply:
Yes. Thanks